not-supported/not-supported_menu.sh

```diff
@@ -17,6 +17,14 @@ debug_mode
 # Must be root
 root_check
 
+# Set the startup switch
+if [ -f "$SCRIPTS/nextcloud-startup-script.sh" ]
+then
+    STARTUP_SWITCH="ON"
+else
+    STARTUP_SWITCH="OFF"
+fi
+
```

Comment:
we only need this as long as smtp mail is in here. I will remove it once it is in server_configuration, again.

Reply:
Sure. :+1: